scheduler/src/trigger-handler.ts

```diff
@@ -1,22 +1,22 @@
-import { AxiosError } from 'axios';
+import { PostgresClient } from '@jvalue/node-dry-pg';
 
-import * as AdapterClient from './api/http/adapter-client';
+import * as EventPublisher from './datasource-trigger/outboxEventPublisher';
 
 export async function triggerDatasource(
+  pgClient: PostgresClient,
   datasourceId: number,
   triggerRetries: number,
 ): Promise<void> {
   for (let i = 0; i < triggerRetries; i++) {
     try {
-      await AdapterClient.triggerDatasource(datasourceId);
+      await pgClient.transaction(
```

Comment:
Not sure about our lib, but do we need a transaction here?

Reply:
I added the transaction, because they were used in other parts, were also only one call was executed, e.g.: 
https://github.com/jvalue/ods/blob/1eef1a7e91ac5b26ae0317329941a1e462385604/pipeline/src/pipeline-config/pipelineConfigManager.ts#L112-L121